spec.md

```diff
@@ -746,6 +746,265 @@ For success responses, the following fields are supported. Others will be ignore
 }
 </pre>
 
+# Service Instance Actions
+These are any actions that a broker allows w.r.t the service instances that it manages. The actions here could be anything that the service broker wants to provide and the platform shouldnt have to care what these are. The actions are discoverable by the platform and since the platform has absolutely no clue what these actions are there would be user interaction involved where the actions are presented to the user by the platform and the user performs the action with any input thats required.
+
+## Action Discovery
+By Implementing this endpoint the broker allows the platform to query the actions allowed on a service instance at any point in time. The list of actions returned by the endpoint is dynamic and could depend on the service instance and its internal state. 
+
+##### Route #####
+`GET /v2/service_instances/:instance_id/actions`
+
+The `:instance_id` of a service instance that has been provisioned previously.
+
+#### cURL ####
+<pre class="terminal">
+ $ curl -H "X-Broker-API-Version: 2.11" http://username:password@broker-url/v2/service_instances/:instance_id/actions
+</pre>
+
+### Response ###
+
+| Status Code  | Description  |
+|---|---|
+| 200 OK  | The expected response body is below. |
+
+##### Action <a name="action"></a> #####
+
+|  Response field | Type  | Description  |
```

Comment:
Right, so the question you're asking from the endpoint is what actions can be performed. I presume each action that you call is an HTTP request and if so, then you're describing an API? Inputs/outputs, etc. and it seems like if you call the /actions, returning a swagger document which is an open spec for describing APIs rather than defining our own format would seem better unless there's a strong reason not to do that?